src/commands/createNewProject/verifyIsProject.ts

```diff
@@ -81,23 +79,14 @@ async function promptForProjectSubpath(context: IActionContext, workspacePath: s
 }
 
 /**
- * Checks if the path is already a function project. If not, it will prompt to create a new project and return undefined
+ * Checks if the path is already a function project, if not return undefined
  */
-export async function verifyAndPromptToCreateProject(context: IActionContext, workspaceFolder: WorkspaceFolder | string, options?: api.ICreateFunctionOptions): Promise<string | undefined> {
-    options = options || {};
-
-    const projectPath: string | undefined = await tryGetFunctionProjectRoot(context, workspaceFolder, 'modalPrompt');
-    if (!projectPath) {
-        if (!options.suppressCreateProjectPrompt) {
-            const message: string = localize('notFunctionApp', 'The selected folder is not a function project. Create new project?');
-            // No need to check result - cancel will throw a UserCancelledError
-            await context.ui.showWarningMessage(message, { modal: true, stepName: 'notAProject' }, DialogResponses.yes);
+export async function verifyProjectPath(context: IActionContext, workspaceFolder?: WorkspaceFolder | string): Promise<string | undefined> {
+    if (workspaceFolder) {
+        const projectPath: string | undefined = await tryGetFunctionProjectRoot(context, workspaceFolder, 'modalPrompt');
+        if (projectPath) {
```

Comment:
nit: You don't need to check `projectPath`, you could just do

`return await tryGetFunctionProjectRoot(context, workspaceFolder, 'modalPrompt');`